include/apq/query.h

```diff
@@ -2,7 +2,11 @@
 
 #include <boost/hana.hpp>
 
+namespace libapq {
+
 template <const char* text, class ... Args>
 auto make_query(Args&& ... args) {
     return boost::hana::make_tuple(std::forward<Args&&>(args) ...);
 }
+
+} // namespace libapq
```

Comment:
You may just delete this file. It had been commited accidentally.

Reply:
Let it be a placeholder :)